Sources/IO/Misc/OBJReader/index.js

```diff
@@ -195,12 +195,12 @@ function vtkOBJReader(publicAPI, model) {
     model.compression = option.compression;
 
     // Fetch metadata
-    return publicAPI.loadData();
+    return publicAPI.loadData({progressCallback: option.progressCallback});
   };
 
   // Fetch the actual data arrays
-  publicAPI.loadData = () => {
-    const promise = fetchData(model.url);
+  publicAPI.loadData = (option = {}) => {
+    const promise = fetchData(model.url, {progressCallback: option.progressCallback});
```

Comment:
This open to other options without loss. Basically you deferred to the dataFetch layer to handle those options.